plugins/treatnet_console/public/services/treatnet_console_api.js

```diff
@@ -50,6 +51,26 @@ class StixAPI extends BaseTreatnetConsoleAPI {
   }
 }
 
+
+class SinkholeAPI extends BaseTreatnetConsoleAPI {
+  get patterns () {
+    return {
+      list: () => this._get('sinkhole/patterns/')
+    };
+  }
+  filterPatterns(params = {}) {
```

Comment:
Wydaje mi się, nie ma potrzeby dodawać ręcznie queryparams do urla w tej metodzie, tylko wystarczy przekazać je w `list: (params) => this._get(url, params)`. Wtedy `filterPatterns` można by zrobić getterem. Jeżeli to nie działa, to lepiej i tak chyba dodać przerabianie queryparams w `this._get` bo jest bazową metodą.